### ISSUE TITLE
OidcClientOptions Help Request

### ISSUE THREAD

**Issue by atulmjoshi** [DuendeArchive/IdentityModel.OidcClient#97]
I am trying to use this library for Okta IdP and using WPF desktop client

In OidcClientOptions
1. I want to specify non-default authorisation server created on okta. Can i use Authority as https://<company>.oktapreview.com/oauth2/ausdhq80q2kjZT8d90h7 ?
2. Does this library supports both PKCE and normal applications?
3. RedirectUri - What would that be if I am using it in desktop wpf application?

Basically I am finding difficult to get answers to above questions - your help is truly appreciated.

**Comment by leastprivilege** [comment]
1. if Okta is using the standard URLs beneath their host - yes
2. PKCE is required
3. many options - check the samples repo

**Comment by leastprivilege** [comment]
OK?

**Comment by atulmjoshi** [comment]
That did not help much. I tried "http://localhost" like urls for handling RedirectUri assuming that its temp address and act as placeholder for receiving code/tonken at WPF application. Is that right? I am also not sure if Okta is supporting standard Urls. 


I will try to get more details in few days.
